src/types.ts

```diff
@@ -52,15 +52,15 @@ interface OSDataHubResponse {
 
 interface ValidationParams extends Options {
   apiKey: string;
-  offset?: number;
-  limit?: number;
   radius?: number;
   point?: [number, number];
   polygon?: Feature | FeatureCollection | Polygon;
   bbox?: [number, number, number, number];
   uprn?: number;
   postcode?: string;
   query?: string;
+  collectionId?: string;
+  featureId?: string;
 }
```

Comment:
I think the validation system at the moment is centralised between ngd, places and names. This is quite handy because of reuse of the bbox validation, for example. But that doesn't mean it's necessarily a good pattern - will have a think